### PR TITLE
fixing ImageStorage_001_deleteImage test by adding redirect=no parameter

### PR DESCRIPTION
**Pull request by justnpT**
https://wikia-inc.atlassian.net/browse/QAART-436

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/imageservingtests/ImageStorageTests.java [hunk 1]
```diff
@@ -34,7 +34,8 @@ public void ImageStorage_001_deleteImage() {
 		WikiBasePageObject base = new WikiBasePageObject(driver);
 		base.logInCookie(credentials.userNameStaff, credentials.passwordStaff, wikiURL);
 		SpecialNewFilesPageObject newFiles = base.openSpecialNewFiles(wikiURL);
-		FilePagePageObject file = newFiles.openImage(PageContent.FILEDELETEANDRESTORE);
+		Boolean noRedirect = true;
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/imageservingtests/ImageStorageTests.java]: no needfor another variable

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 2]
```diff
@@ -135,8 +137,14 @@ public String getImageUrl(String imageName) {
 		throw new NoSuchElementException("there is no " + imageName + " on Special:NewFiles page");
 	}
 
-	public FilePagePageObject openImage(String imageName) {
-		driver.get(getImageUrl(imageName));
+	public FilePagePageObject openImage(String imageName, boolean noRedirect) {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: Add JAVAdoc

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 3]
```diff
@@ -135,8 +137,14 @@ public String getImageUrl(String imageName) {
...
+		String url = getImageUrl(imageName);
+		if (noRedirect) {
+			String parameter = "redirect=no";
+			UrlBuilder builer = new UrlBuilder();
+			url = builer.appendQueryStringToURL(url, parameter);
+		}
+		driver.get(url);
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: builder object is allready there - in base ....

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 4]
```diff
@@ -135,8 +137,14 @@ public String getImageUrl(String imageName) {
 		throw new NoSuchElementException("there is no " + imageName + " on Special:NewFiles page");
 	}
 
-	public FilePagePageObject openImage(String imageName) {
-		driver.get(getImageUrl(imageName));
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: You can overload insted of deleting + mark old method as depricated

**Comment by justnpT** [comment]
all notes adressed

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**